adcc/Excitation.py

```diff
@@ -72,7 +72,13 @@ def __init__(self, parent_state, index, method):
             kwargs = getattr(fget, "__excitation_property").copy()
 
             def get_parent_property(self, key=key, kwargs=kwargs):
-                return getattr(self.parent_state, key)[self.index]
+                if callable(getattr(self.parent_state, key)):
+                    def wrapper(*args, **kwargs):
+                        callback = getattr(self.parent_state, key)
+                        return callback(*args, **kwargs)[self.index]
+                    return wrapper
+                else:
+                    return getattr(self.parent_state, key)[self.index]
```

Comment:
I think these changes are not necessary, because `ExcitedStates.excitation_property_keys` filters out all class methods that are not decorated with the `property` decorator anyway. So it is only possible to trigger the `else` clause.